test/service/test_response.py

```diff
@@ -1251,8 +1248,33 @@ class TestResponseSummary(WebServiceTestCase):
     maxDiff = None
     bundles = WebServiceTestCase.bundles + [
         ('dcccb551-4766-4210-966c-f9ee25d19190', '2018-10-18T204655.866661Z'),
+        ('94f2ba52-30c8-4de0-a78e-f95a3f8deb9c', '2019-04-03T103426.471000Z')  # an imaging bundle
     ]
 
+    def test_summary_response(self):
+        """
+        Verify the /repository/summary response with the following indexed bundles:
+        - bundle=aaa96233, fileCount=2, donorCount=1, totalCellCount=1.0, organType=pancreas
+        - bundle=dcccb551, fileCount=19, donorCount=4, totalCellCount=6210.0, organType=Brain
+        - bundle=94f2ba52, fileCount=227, donorCount=1, totalCellCount=0, organType=brain
+        """
+        url = self.base_url + "/repository/summary"
+        response = requests.get(url)
+        response.raise_for_status()
+        summary_object = response.json()
+        self.assertEqual(summary_object['fileCount'], 2 + 19 + 227)
+        self.assertEqual(summary_object['donorCount'], 1 + 4 + 1)
+        self.assertEqual(summary_object['totalCellCount'], 1.0 + 6210.0 + 0)
+        file_counts_expected = {'tiff': 221, 'json': 6, 'fastq.gz': 5, 'tsv': 4, 'h5': 3,
+                                'pdf': 3, 'mtx': 2, 'bai': 1, 'bam': 1, 'csv': 1, 'unknown': 1}
+        file_counts_actual = {summary['fileType']: summary['count'] for summary in summary_object['fileTypeSummaries']}
+        self.assertEqual(file_counts_actual, file_counts_expected)
+        self.assertEqual(set(summary_object['organTypes']), {'Brain', 'brain', 'pancreas'})
+        self.assertEqual(summary_object['cellCountSummaries'], [
+            {'organType': ['Brain'], 'countOfDocsWithOrganType': 1, 'totalCellCountByOrgan': 6210.0},
+            {'organType': ['pancreas'], 'countOfDocsWithOrganType': 1, 'totalCellCountByOrgan': 1.0},
+        ])
```

Comment:
Explain in a comment why "brain" doesn't occur.

Reply:
Added explanation in comment